sw/otbn/crypto/p384_isoncurve.s

```diff
@@ -16,6 +16,40 @@
 
  .section .text
 
+/**
+ * Trigger a fault if the FG0.Z flag is 0.
+ *
+ * If the flag is 0, then this routine will trigger an `ILLEGAL_INSN` error and
+ * abort the OTBN program. If the flag is 1, the routine will essentially do
+ * nothing.
+ *
+ * NOTE: Be careful when calling this routine that the FG0.Z flag is not/**
+ * Trigger a fault if the FG0.Z flag is 0.
+ *
+ * @param[in]    w31: all-zero
+ * @param[in]  FG0.Z: boolean indicating fault condition
+ *
+ * clobbered registers: x2
+ * clobbered flag groups: none
+ */
+ .globl trigger_fault_if_fg0_not_z
```

Comment:
do we have a coding style for where to put the .globl ? In other files we put them at the top